plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -107,11 +108,56 @@ tests =
     ]
   , goldenWithEval "Works with NoImplicitPrelude" "TNoImplicitPrelude" "hs"
   , goldenWithEval "Variable 'it' works" "TIt" "hs"
+
+  , testGroup ":info command"
+    [ testCase ":info reports type, constructors and instances" $ do
```

Comment:
As suggested by @jneira, these tests now only look for the relevant infix strings (or their absence)